packages/core-components/src/layout/Sidebar/Items.tsx

```diff
@@ -702,7 +702,7 @@ export const SidebarScrollWrapper = styled('div')(({ theme }) => {
     flex: '0 1 auto',
     overflowX: 'hidden',
     // 5px space to the right of the scrollbar
-    width: 'calc(100% - 5px)',
+    width: 'calc(100%)',
```

Comment:
should probably remove the calc and also the comment above right?

Reply:
You are absolutely right. I'll take care of it this evening.